styles/pronouns.cc/catppuccin.user.css

```diff
@@ -0,0 +1,266 @@
+      --bs-list-group-active-border-color: @accent-color;
+      border-radius: var(--bs-list-group-border-radius);
+    }
+    // navbar
+    .navbar {
+      --bs-navbar-color: @subtext0;
+      --bs-navbar-hover-color: @subtext0;
+      --bs-navbar-disabled-color: @subtext1;
+      --bs-navbar-active-color: @text !important;
+      --bs-navbar-brand-color: @text !important;
+      --bs-navbar-brand-hover-color: @text !important;
+      --bs-navbar-toggler-border-color: @surface0;
+    }
+    #colorizedButton(@color, @bg, @border: @bg, @hv-color: @color, @hv-bg: darken(@bg, 5%), @hv-border: @hv-bg) {
+      --bs-btn-color: @color;
+      --bs-btn-bg: @bg;
+      --bs-btn-border-color: @border;
+
+      --bs-btn-hover-color: @hv-color;
+      --bs-btn-hover-bg: @hv-bg;
+      --bs-btn-hover-border-color: @hv-border;
+
+      --bs-btn-active-color: @hv-color;
+      --bs-btn-active-bg: @hv-bg;
+      --bs-btn-active-border-color: @hv-border;
+
+      --bs-btn-focus-shadow-rgb: #rgbify(@bg) [];
+      --bs-btn-disabled-color: @color;
+      --bs-btn-disabled-bg: @bg;
+      --bs-btn-disabled-border-color: @border;
+    }
+    #coloredButton(@color) {
+      #colorizedButton(@crust, @color, @color, @hv-color: @crust, lighten(@color, 10%));
+    }
+    //buttons
+    .btn-primary {
+      #colorizedButton(@base, @accent-color, @hv-bg: lighten(@accent-color, 10%));
+    }
+    .btn-secondary {
+      #colorizedButton(@text, @surface0, @hv-bg: @surface1);
+    }
+    .btn-outline-secondary {
+      #colorizedButton(@surface1, transparent, @surface1, @subtext0, @surface1);
+    }
+    .btn-success {
+      #coloredButton(@green);
+    }
+    .btn-info {
+      #coloredButton(@teal);
+    }
+    .btn-warning {
+      #coloredButton(@yellow);
+    }
+    .btn-danger {
+      #coloredButton(@red);
+    }
+    .form-control {
+      &::-webkit-file-upload-button,
+      &::file-selector-button {
+        background-color: @base;
+      }
+      &:hover:not(:disabled, [readonly]) {
+        &::-webkit-file-upload-button,
+        &::file-selector-button {
+          background-color: @surface0;
+        }
+      }
+    }
+    .btn-close-white,
+    .btn-close {
+      --bs-btn-close-color: @surface0;
+      @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill="@{surface1}"><path d="M.293.293a1 1 0 0 1 1.414 0L8 6.586 14.293.293a1 1 0 1 1 1.414 1.414L9.414 8l6.293 6.293a1 1 0 0 1-1.414 1.414L8 9.414l-6.293 6.293a1 1 0 0 1-1.414-1.414L6.586 8 .293 1.707a1 1 0 0 1 0-1.414"/></svg>');
+      --bs-btn-close-bg: url("data:image/svg+xml,@{svg}");
+      --bs-btn-close-opacity: 1.0;
+      --bs-btn-close-hover-opacity: .75;
+      --bs-btn-close-focus-shadow: 0 0 0 .25rem rgba(13, 110, 253, .25);
+      --bs-btn-close-focus-opacity: 1;
+      --bs-btn-close-disabled-opacity: .25;
+      --bs-btn-close-white-filter: none;
+      color: var(--bs-btn-close-color);
+      opacity: var(--bs-btn-close-opacity);
```

Comment:
Do we need to set these?
```suggestion
```

Reply:
we actually need the first line but the rest can go.